src/System.Private.ServiceModel/tests/Common/Infrastructure/tests/project.json

```diff
@@ -5,11 +5,21 @@
     "System.ServiceModel.Primitives": "4.1.0-rc3-23931-00",
     "System.ServiceModel.Security": "4.0.1-rc3-23931-00",
     "xunit": "2.1.0",
-    "xunit.netcore.extensions": "1.0.0-prerelease-00231-07"
+    "xunit.netcore.extensions": "1.0.0-prerelease-00231-07",
+    "test-runtime": {
+      "target": "project",
+      "exclude": "compile"
+    }
   },
   "frameworks": {
     "dnxcore50": {
       "imports": "portable-net45+win8"
     }
+  },
+  "runtimes": {
+    "win7-x64": {},
+    "win7-x86": {},
+    "ubuntu.14.04-x64": {},
+    "osx.10.10-x64": {}
   }
```

Comment:
For projects that are going to be published for runtime usage, we need to list all of the runtimes they may be used on. Currently, we are relying on the "--infer-runtimes" feature of the CLI, which creates a lock file for your current runtime (that is my understanding of it, anyways). That feature is going away, so we will need to list all of the runtimes regardless.